lib/live_debugger/live_views/live_views_dashboard_live.ex

```diff
@@ -25,7 +25,7 @@ defmodule LiveDebugger.LiveViews.LiveViewsDashboardLive do
       <div class="max-lg:p-8 pt-8 w-full lg:max-w-[40rem] h-full">
         <div class="flex gap-4 items-center justify-between">
           <.h1>Active LiveViews</.h1>
-          <.button phx-click="refresh">
+          <.button id="refresh" phx-click="refresh">
```

Comment:
I wouldn't add this type ids to production code only to have easier way to write selectors in test. FYI you can write query selector like this to query this button `button[phx-click="refresh"]`